python/metatensor-torch/metatensor/torch/atomistic/documentation.py

```diff
@@ -144,6 +144,8 @@ def add_data(self, name: str, data: TensorBlock):
 
         :param name: name of the custom data
         :param data: values of the custom data
+        :param override: if true, allow this function to override existing data with the
```

Comment:
I don't know if we follow this but we could link to the Python docs here.

```suggestion
        :param override: if :py:obj:`True`, allow this function to override existing data with the
```

Reply:
TBH, I'm not sure what we (or the users) gain by linking to the docs of `True`/`False`/`None`. Happy to put that inside backticks for a better rendering